targets/atmega1280.json

```diff
@@ -0,0 +1,17 @@
+{
+    "inherits": ["avr"],
+    "cpu": "atmega1280",
+    "build-tags": ["atmega1280", "atmega"],
+    "cflags": [
+        "-mmcu=atmega1280"
+    ],
+    "ldflags": [
+        "-mmcu=avr5",
```

Comment:
According to https://gcc.gnu.org/onlinedocs/gcc/AVR-Options.html, this should be `-mmcu=avr51`.